modules/regex/regex.cpp

```diff
@@ -321,7 +321,7 @@ struct RegExNodeClass : public RegExNode {
 			case Type_lower:
 				return ('a' <= c && c <= 'z');
 			case Type_print:
-				return (0x1F < c && c < 0x1F);
+				return (0x1F < c && c < 0x1F);	// huh ?
```

Comment:
Ahhh, it's supposed to be `7F`. Also, referring to the POSIX standards again, space is excluded too so it should actually be `0x20 < c && c < 0x7F`. 😅